protoc-gen-pbandk/lib/src/commonMain/kotlin/pbandk/gen/FileBuilder.kt

```diff
@@ -239,11 +239,43 @@ internal open class FileBuilder(val namer: Namer = Namer.Standard, val supportMa
             ?.associate { it.substringBefore("->") to it.substringAfter("->", "") }
             ?: emptyMap()
 
+        // Convert each mapping to a regex pattern, and then associate it with the replacement string
+        val packageMappingRegexes = packageMappings
+            .filter { it.value.contains("*") || it.value.contains("?") }
```

Comment:
Why the check for the `?` character? It doesn't look like the code you added does anything with that.

Reply:
I was thinking about including `?` which is a single-character match in bash but I abandoned that idea. Let me revert that.